libs/langchain-community/src/langgraph_checkpointers/vercel_kv.ts

```diff
@@ -0,0 +1,153 @@
+import { VercelKV, createClient } from "@vercel/kv";
+
+import { RunnableConfig } from "@langchain/core/runnables";
+import {
+  BaseCheckpointSaver,
+  Checkpoint,
+  CheckpointMetadata,
+  CheckpointTuple,
+} from "@langchain/langgraph";
+
+import { SerializerProtocol } from "../serde/base.js";
```

Comment:
Should import from `@langchain/langgraph/web` as well

Reply:
👍